Gemfile.lock

```diff
@@ -134,9 +135,10 @@ PLATFORMS
   x86_64-linux
 
 DEPENDENCIES
+  benchmark-ips
   google-cloud-bigtable
   google-protobuf
   protoboeuf!
 
 BUNDLED WITH
-   2.5.22
+   2.6.3
```

Comment:
I really dislike these changes. It's not your fault, but Bundler shouldn't be upgrading itself if it's not needed.

Reply:
I did it explicitly because when I installed stuff the first time it tried to remove a bunch of platform specific items which then made the installation compile stuff on my linux pc.
It didn't happen again after upgrading bundler.  Not sure if there was another issue

Reply:
I set the bundler version back, then when I tried `bundle exec` it told me I didn't have a gem installed...
This version of bundler (2.5.22) seems to want to discard the platform gems and compile the gem itself.
With 2.6.3 it seems to handle that differently 🤷

Reply:
I'm sorry. You didn't need to change it back. I was just lamenting the fact that Bundler indiscriminately makes these sorts of modifications so they get added to otherwise unrelated changesets.

Reply:
If I set my local ruby to 3.3.6 it stops doing that.  Something about the mismatch of newer ruby and older bundler (I was on 3.4.1).